### PR TITLE
onboard USDe - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/AaveV3Ethereum_OnboardUSDeAaveV3Ethereum_20240528.sol [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      borrowableInIsolation: EngineFlags.DISABLED,
+      withSiloedBorrowing: EngineFlags.DISABLED,
+      flashloanable: EngineFlags.ENABLED,
+      ltv: 72_00,
+      liqThreshold: 75_00,
+      liqBonus: 8_50,
+      reserveFactor: 25_00,
+      supplyCap: 80_000_000,
+      borrowCap: 72_000_000,
+      debtCeiling: 0,
```
Comment (kyzia551) [inline src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/AaveV3Ethereum_OnboardUSDeAaveV3Ethereum_20240528.sol:47]: 40M in[ forum post](https://governance.aave.com/t/arfc-onboard-usde-to-aave-v3-on-ethereum/17690)

File: src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/OnboardUSDeAaveV3Ethereum.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+The table below illustrates the configured risk parameters for **USDe**
+
+| Parameter                          |                                      Value |
+| ---------------------------------- | -----------------------------------------: |
+| Isolation Mode                     |                                       true |
+| Borrowable                         |                                    ENABLED |
+| Collateral Enabled                 |                                       true |
+| Supply Cap (USDe)                  |                                 80,000,000 |
+| Borrow Cap (USDe)                  |                                 72,000,000 |
+| Debt Ceiling                       |                                      USD 0 |
```
Comment (kyzia551) [inline src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/OnboardUSDeAaveV3Ethereum.md:27]: 40M in[ forum post](https://governance.aave.com/t/arfc-onboard-usde-to-aave-v3-on-ethereum/17690)

**Review by kyzia551 — CHANGES_REQUESTED** [review]

File: src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/AaveV3Ethereum_OnboardUSDeAaveV3Ethereum_20240528.sol [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    IAaveV3ConfigEngine.Listing[] memory listings = new IAaveV3ConfigEngine.Listing[](1);
+
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: USDe,
+      assetSymbol: 'USDe',
+      priceFeed: 0x55B6C4D3E8A27b8A1F5a263321b602e0fdEEcC17,
+      eModeCategory: AaveV3EthereumEModes.NONE,
+      enabledToBorrow: EngineFlags.ENABLED,
+      stableRateModeEnabled: EngineFlags.DISABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
```
Comment (rustboyar) [inline src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/AaveV3Ethereum_OnboardUSDeAaveV3Ethereum_20240528.sol:38]: should it be true as Isolation mode is enabled?

File: src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/AaveV3Ethereum_OnboardUSDeAaveV3Ethereum_20240528.sol (hunk 1 above)
Comment (marczeller) [reply src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/AaveV3Ethereum_OnboardUSDeAaveV3Ethereum_20240528.sol]: fixed

File: src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/AaveV3Ethereum_OnboardUSDeAaveV3Ethereum_20240528.sol (hunk 3 above)
Comment (marczeller) [reply src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/AaveV3Ethereum_OnboardUSDeAaveV3Ethereum_20240528.sol]: fixed

**Review by kyzia551 — APPROVED** [review]

File: src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/AaveV3Ethereum_OnboardUSDeAaveV3Ethereum_20240528.sol (hunk 3 above)
Comment (Rozengarden) [reply src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/AaveV3Ethereum_OnboardUSDeAaveV3Ethereum_20240528.sol]: am not sure, if I rember correctly, isolation mode means that an asset have a debt limit and can only be used to borrow a limited subset of assets. Setting a debt ceiling to something else than 0 activate isolation mode.  While borrowable in isolation mode is used to add an asset to the previously mentioned borrowable subset.

**Review by rustboyar — APPROVED** [review]

**Pull request merged**